rose-admin

```diff
@@ -4,12 +4,7 @@ import sys
 import os
 import logging
 
-import six
-
-if six.PY2:
-    import xmlrpclib
-else:
-    from xmlrpc import client as xmlrpclib
+from xmlrpc import client as xmlrpclib
```

Comment:
This is unrelated change (remove python 2 leftovers) best done in a separate commit.

Reply:
Thanks, created a python2 cleanup PR #478

Reply:
Removed the python2 cleanup code in favor or #478